### ISSUE TITLE
Error When Deploying Streamlit App Using Snowcli

### ISSUE THREAD

**Issue by mashhype** [Snowflake-Labs/snowcli#495]
### SnowCLI version

2.0.0-dev

### Python version

3.9.6

### Platform

macOS-13.5.2-arm64-arm-64bit

### What happened

when I run the following command in snowcli:
snow streamlit deploy demo_app --file=Home.py -c mashsf  --warehouse MED_COMPUTE_WH

### Console output

```Shell
Usage: snow streamlit deploy [OPTIONS]
Try 'snow streamlit deploy --help' for help.
╭─ Error ────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ No such option: --file Did you mean --role?
```

### How to reproduce

Not sure

**Comment by sfc-gh-astus** [comment]
We changed streamlit commands, most of flags was moved to yaml file. We don't have documentation yet. Please use 1.2.0-rc7 version

**Comment by mashhype** [comment]
I instlled rc7, then ran same command and now I get this:

<img width="925" alt="Screenshot 2023-10-26 at 9 52 02 AM" src="https://github.com/Snowflake-Labs/snowcli/assets/4706698/ef534d45-856d-4feb-b5a7-ce9718c4bdb2">

**Comment by mashhype** [comment]
got this command to work when using the bash terminal, issue is isolated to using the terminal inside Jupyter Lab

**Comment by sfc-gh-astus** [comment]
Did you write that command or copy it? There is no typo and I'm not able to reproduce it

**Comment by mashhype** [comment]
In the terminal inside Jupyter Lab, I did both, copied and typed it out. When I got it working in the bash terminal, I believe I copied as well and changed one of the arguments, and it worked fine.

**Comment by sfc-gh-astus** [comment]
Hi @mashhype, is it working or there is still some issue?

**Comment by mashhype** [comment]
Yes its working in both bash terminal and Jupyter Lab terminal. But to make it work, I cannot simply copy and paste the command from another application (Word, Notes). I have to type the argument from scratch.
